python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_path_selection.schema.yml

```diff
@@ -15,6 +15,19 @@ keys:
         valid_values:
           - stun
         description: Source of dynamic peer discovery.
+      mtu_discovery:
+        type: dict
+        keys:
+          enabled:
```

Comment:
I think this should be `hosts` to align with CLI and then in the description we can mention it is to enable MTU discovery for hosts on the LAN.